src/app/modules/level-select/level-select.module copy.ts

```diff
@@ -0,0 +1,14 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+
+import { LevelSelectRoutingModule } from './level-select-routing.module';
+
+
+@NgModule({
+  declarations: [],
+  imports: [
+    CommonModule,
+    LevelSelectRoutingModule
+  ]
+})
+export class LevelSelectModule { }
```

Comment:
What this file for?)

Reply:
I don't know it was by default

Reply:
Should I remove it?

Reply:
it was not there by default:))

Reply:
Originally in the folder modules/level-select there were 2 files level-select-routing.module.ts and level-select.module.ts

Reply:
yep, but we are talking about "level-select.module copy.ts"

Reply:
my fault